apps/mobile/src/libs/components/wheel-loader/wheel-loader.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+import { Wheel } from "~/libs/components/components";
+import { AnimationName, GradientColor } from "~/libs/enums/enums";
+
+type Properties = {
+	animationDuration: number;
+	size: number;
+};
+
+const mockData = [
```

Comment:
Let's name it gradientFragmentsData

Reply:
May be better gradientSectorsData? I have referred to the parts of chart as sectors already.